webroot/src/pages/FinalizePrivilegePurchase/FinalizePrivilegePurchase.vue

```diff
@@ -0,0 +1,171 @@
+                                />
+                            </div>
+                            <div class="form-row">
+                                <InputText
+                                    :formInput="formData.streetAddress2"
+                                />
+                            </div>
+                            <div class="state-zip-line form-row">
+                                <InputSelect
+                                    :formInput="formData.stateSelect"
+                                />
+                                <InputText
+                                    :formInput="formData.zip"
+                                    @input="handleZipInput(formData.cvv)"
+                                />
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="cost-breakdown-container">
+                    <div class="cost-listing-block">
+                        <div class="selection-title">{{selectionText}}</div>
+                        <ul>
+                            <li
+                                v-for="(state) in seletedStatePurchaseDsiplayDataList"
+                                :key="state.jurisdiction.abbrev"
+                                class="selected-state-block"
+                            >
+                                <div class="info-row">
+                                    <div class="info-row-label">
+                                        {{state.stateFeeText}}
+                                    </div>
+                                    <div class="expire-date-value">${{state?.fee?.toFixed(2)}}</div>
+                                </div>
+                                <div v-if="state.isMilitaryDiscountActive" class="info-row">
+                                    <div class="info-row-label">
+                                        {{state.stateMilitaryPurchaseText}}
+                                    </div>
+                                    <div class="expire-date-value">-${{state?.militaryDiscountAmount?.toFixed(2)}}</div>
+                                </div>
+                            </li>
+                        </ul>
+                        <div class="compact-commission-fee info-row">
+                            <div class="info-row-label">
+                                {{compactCommissionFeeText}}
+                            </div>
+                            <div class="expire-date-value">${{totalCompactCommissionFee?.toFixed(2)}}</div>
+                        </div>
+                        <div class="purchase-total info-row">
+                            <div class="info-row-label total">
+                                {{totalTitle}}
+                            </div>
+                            <div class="expire-date-value total">${{totalPurchasePrice?.toFixed(2)}}</div>
```

Comment:
I think I'd prefer to handle these template values `{{}}` discretely, as in handling the existence & formatting in the computed props, rather than the template. My main reasons are 1) consistency of handling the calcs in the code and not in the template, and 2) using the optional chaining operator `?.` breaks the HTML formatting here in github and also in the couple editors I tested with - which isn't as critical but does suggest a potential overreach of JS code into the HTML templates.